recipes/cibots/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+    name: cibots
+    version: 0.1
+
+source:
+    url: https://github.com/UV-CDAT/ci-bots/archive/0.1.tar.gz
+    sha256: c16d96619efa86fdabc23f49b6bff04a78e1d99a097a5fa8320504038dd6976b
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install
```

Comment:
hum...
python setup.py install --single-version-externally-managed --record record.txt
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option --single-version-externally-managed not recognized